src/components/EntityInfo/EntityDefinitions.vue

```diff
@@ -143,6 +153,7 @@ export default class EntityDefinitions extends Vue {
 dl {
   font-size: $px-14;
   line-height: 1.5rem;
+  margin-top: 0.2rem;
```

Comment:
This adds whitespace to the top of this component. The parent should set this (not here), so that different parents could specify different amounts of top whitespace.

Reply:
Fixed, thanks.